README.md

```diff
@@ -1,21 +1,32 @@
-![Aquifer](https://raw.githubusercontent.com/aquifer/aquifer.io/master/branding/aquifer-logo-drupal-1456.png)
+![Aquifer][logo]
```

Comment:
Basically I reformatted the readme to be more viewable by terminal editors and hard wraps at 80 chars. By using the alternate version of links and images, you can place their URLS in a list at the bottom of a section making them more organized and allowing the text itself to wrap easier. See [Link Markdown documentation](https://daringfireball.net/projects/markdown/syntax#link)